soco/services.py

```diff
@@ -553,7 +690,15 @@ def iter_actions(self):
             statevars = srvStateTable.findall('{}stateVariable'.format(ns))
             for state in statevars:
                 name = state.findtext('{}name'.format(ns))
-                vartypes[name] = state.findtext('{}dataType'.format(ns))
+                datatype = state.findtext('{}dataType'.format(ns))
+                default = state.findtext('{}defaultValue'.format(ns))
+                vlist = [item.text for item in
```

Comment:
although the rest of this method has lots of aggressively abbreviated variables, I'd probably go with `value_list` and `value_range`

Reply:
Okay, and I'll also decompress those list comprehensions a bit.